Documentation/v2/admin_guide.md

```diff
@@ -165,13 +165,16 @@ $ ssh 10.0.1.11
 ```
 
 ```sh
-$ kill `pgrep etcd`
+$ systemctl stop etcd2
```

Comment:
The kill example is highly problematic because any sort of process monitor (one of which will be quite commonly used to run etcd) is going to re-launch it after you kill it like this.

This systemctl command at least works on a CoreOS host, which the `kill ...` one does not.

I would personally prefer just having the `systemctl` one but mentioning something like "...if running etcd under systemd, such as CoreOS does. If you are running it under a different process monitor, use that system's stop command. If you are sure you aren't running it under any kind of process monitor, `kill \`pgrep etcd\`` might work."

What do you think?